<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1502</width>
    <height>785</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="4">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,9">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>原图片</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="VideoWidget" name="lab_face">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>480</width>
                    <height>480</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QLabel {
    border: 2px solid black;
    padding: 2px; /* 如果需要增加内边距，可选 */
}
</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,9">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>目标检测</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="VideoWidget" name="lab_dectface">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>480</width>
                    <height>480</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QLabel {
    border: 2px solid black;
    padding: 2px; /* 如果需要增加内边距，可选 */
}
</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,5,1">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>检测结果</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="VideoWidget" name="lab_dectarea">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>200</width>
                    <height>200</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QLabel {
    border: 2px solid black;
    padding: 2px; /* 如果需要增加内边距，可选 */
}
</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_2" native="true">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>150</height>
              </size>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1">
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="title">
                 <string>脸部遮罩（摄像头）</string>
                </property>
                <widget class="QPushButton" name="btn_mask">
                 <property name="geometry">
                  <rect>
                   <x>40</x>
                   <y>40</y>
                   <width>75</width>
                   <height>24</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>图片1</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="btn_mask2">
                 <property name="geometry">
                  <rect>
                   <x>130</x>
                   <y>40</y>
                   <width>75</width>
                   <height>24</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>图片2</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="btn_mask3">
                 <property name="geometry">
                  <rect>
                   <x>220</x>
                   <y>40</y>
                   <width>75</width>
                   <height>24</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>图片3</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="btn_none_mask">
                 <property name="geometry">
                  <rect>
                   <x>310</x>
                   <y>40</y>
                   <width>75</width>
                   <height>24</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>无遮罩</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="btn_save">
                 <property name="geometry">
                  <rect>
                   <x>300</x>
                   <y>80</y>
                   <width>131</width>
                   <height>24</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>保存检测图片</string>
                 </property>
                </widget>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_3">
                <property name="title">
                 <string>部位检测</string>
                </property>
                <widget class="QRadioButton" name="cbx_eyes">
                 <property name="geometry">
                  <rect>
                   <x>110</x>
                   <y>20</y>
                   <width>91</width>
                   <height>16</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>眼睛</string>
                 </property>
                </widget>
                <widget class="QRadioButton" name="cbx_face">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>25</y>
                   <width>61</width>
                   <height>16</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>脸部</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
                <widget class="QRadioButton" name="cbx_mouth">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>100</y>
                   <width>71</width>
                   <height>16</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>嘴巴</string>
                 </property>
                </widget>
                <widget class="QRadioButton" name="cbx_contours">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>60</y>
                   <width>61</width>
                   <height>16</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>轮廓</string>
                 </property>
                </widget>
                <widget class="QRadioButton" name="cbx_nose">
                 <property name="geometry">
                  <rect>
                   <x>110</x>
                   <y>60</y>
                   <width>61</width>
                   <height>16</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>鼻子</string>
                 </property>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1502</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_open_picture"/>
    <addaction name="action_open_video"/>
    <addaction name="action_open_camera"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_open_picture">
   <property name="text">
    <string>打开图片</string>
   </property>
  </action>
  <action name="action_open_video">
   <property name="text">
    <string>打开视频</string>
   </property>
  </action>
  <action name="action_open_camera">
   <property name="text">
    <string>打开摄像头</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QLabel</extends>
   <header location="global">video_widget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
